keyboards/keebfront/vanguard65/config.h

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 <hello@keebfront.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Define MIDI */
+#define MIDI_ADVANCED
+#define SLIDER_PINA D6
```

Comment:
This does not appear to be used anywhere?

Reply:
@waffle87 This is actually used extensively in the MIDI definition, but the features are not present in the `default` keymap as I understand QMK wants to have it be clean. 

Is there a different way to define this so the lines aren't in QMKs repo but still will activate when compiling for VIA?

Reply:
If there is a slider on the board, the expectation would be for code to exist to control its behaviour.

"clean" doesnt mean zero code in the default keymap.

Reply:
Oh I see the confusion. Last time I understood it to mean "nothing besides a keymap" in there. 

I'll add the basic MIDI functionality in (which in my view is the core), but there is a huge codebase involved for the fully-fledged functionality which is inseparable from VIA's web UI. Guessing you guys don't want all of that code-base though.